scripts/mujoco/create_dataset.py

```diff
@@ -0,0 +1,115 @@
+import gymnasium as gym
+import minari
+from huggingface_sb3 import load_from_hub
+from stable_baselines3 import PPO, SAC, TD3
+from tqdm import tqdm
+
+ENV_IDS = [
+    ("InvertedPendulum", ("medium", "expert"), 100_000, "SAC"),
+    ("InvertedDoublePendulum", ("medium", "expert"), 100_000, "SAC"),
+    ("Reacher", ("medium", "expert"), 500_000, "SAC"),
+    ("Pusher", ("medium", "expert"), 500_000, "SAC"),
+    ("HalfCheetah", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Hopper", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Walker2d", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Swimmer", ("expert"), 1_000_000, "SAC"),
+    ("Ant", ("simple", "medium"), 1_000_000, "SAC"),
+    ("Humanoid", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    #("HumanoidStandup", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+]
+
+
+def create_dataset_from_policy(dataset_id, collector_env, policy, n_steps: int, algorithm_name):
+    truncated = True
+    terminated = True
+    seed = 123
+    for step in tqdm(range(n_steps)):
+        if terminated or truncated:
+            obs, _ = env.reset(seed=seed)
+            seed += 1
+            if (n_steps - step) < collector_env.spec.max_episode_steps :  # trim trailing non-full episodes
+                break
+
+        action = policy(obs)
+        obs, _, terminated, truncated, _ = env.step(action)
+
+    return collector_env.create_dataset(
+        dataset_id=f"mujoco/{dataset_id}",
+        algorithm_name="SB3/{algorithm_name}",
+        code_permalink="https://github.com/Farama-Foundation/minari-dataset-generation-scripts",
+        author="Kallinteris Andreas",
+        author_email="kallinteris@protonmail.com",
+        requirements=[
+            "mujoco==3.2.3",
```

Comment:
you may want to add a description for each dataset. These are encoded in markdown in the documentation.

For readability, you can create a folder with all the description files `.md` for each dataset, and here read from the file based on dataset_id